Cork/Localizable.xcstrings

```diff
@@ -14236,6 +14272,12 @@
             "state" : "needs_review",
             "value" : "error.package-loading.%@-does-not-have-any-versions-installed"
           }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "并未安装软件包%@的任何版本"
```

Comment:
Please leave a blank between %@ and the Chinese characters as the name of the packages are usually in English